modules/database/sqlite/wp-includes/sqlite/class-wp-sqlite-db.php

```diff
@@ -2,23 +2,23 @@
 /**
  * Extend and replace the wpdb class.
  *
- * @package performance-lab
- * @since 1.8.0
+ * @package wp-sqlite-integration
+ * @since 1.0.0
```

Comment:
Since at this point this is still primarily a module, it would be great to keep this using `performance-lab` and the relevant version numbers. You can probably do a search-replace for that.

Our build process will automatically replace this with a plugin-specific package name when building the standalone plugin from the module.

Once we have removed the module and it's only a standalone plugin, we can generally update it.